plugins/rbac/src/components/CreateRole/CreateRoleForm.tsx

```diff
@@ -17,35 +17,70 @@ import { Alert } from '@material-ui/lab';
 import { FormikHelpers, useFormik } from 'formik';
 
 import { rbacApiRef } from '../../api/RBACBackendClient';
+import { MemberEntity } from '../../types';
 import { getRoleData, validationSchema } from '../../utils/create-role-utils';
+import { getKindNamespaceName } from '../../utils/rbac-utils';
+import { useToast } from '../ToastContext';
 import { AddedMembersTable } from './AddedMembersTable';
 import { AddMembersForm } from './AddMembersForm';
 import { RoleDetailsForm } from './RoleDetailsForm';
-import { CreateRoleFormValues } from './types';
+import { CreateRoleFormValues, SelectedMember } from './types';
 
-export const CreateRoleForm = () => {
-  const [activeStep, setActiveStep] = React.useState<number>(0);
+type CreateRoleFormProps = {
+  roleName: string;
+  step?: number;
+  members: { members: MemberEntity[]; loading: boolean; error: Error };
```

Comment:
nit: naming suggestion

```suggestion
  membersData: { members: MemberEntity[]; loading: boolean; error: Error };
```